### PR TITLE
Quran.com Data

### LINKED ISSUE #13: New datastet
Download the https://api.qurancdn.com/api/qdc/audio/reciters?locale=en&fields=undefined.

Download all data, including all metadata.

### PR DESCRIPTION
**Pull request by abdoei**
Closes #13

### CODE REVIEW

**Comment by abdoei** [comment]
I have PRed on this branch as it is the most recent one

**Comment by AbdulrhmnGhanem** [comment]
Could you please rebase it?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.vscode
```
Comment (AbdulrhmnGhanem) [inline .gitignore:1]: Why are ignoring it?

File: experiments/notebooks.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Significant Notebooks Of The Project
```
Comment (AbdulrhmnGhanem) [inline experiments/notebooks.md:1]: Add an empty line at the end.

**Review by AbdulrhmnGhanem — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (abdoei) [reply .gitignore:1]: I use other extensions that adds to it that is irrelevant to other contributors

**Review by AbdulrhmnGhanem — APPROVED** [review]

**Pull request merged**